specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -3334,46 +3354,48 @@
         ],
         "responses": {
           "200": {
-            "description": "Successfully deleted hostname binding."
+            "description": "Function state returned.",
+            "schema": {
+              "type": "string"
```

Comment:
This is a malformed response body. From the URI, "state" is the name of a resource type "properties."  This should have standard proxy resource modeling: id, type, name, properties. However, I don't understand why this isn't a property on the parent resource with PATCH support to toggle state.

Reply:
Yes, you're right. The response returned from the API is of the form:

```javascript
{
  "id": "/subscriptions/de6db429-0822-4034-8456-fccf4deb3841/resourceGroups/Default-Web-WestUS/providers/Microsoft.Web/sites/mathewc-v2-test/functions/httptriggertest/properties/state",
  "name": "state",
  "type": "Microsoft.Web/sites/functions/properties",
  "location": "West US",
  "properties": "enabled"
}
```

It's an ARM wrapped response with the content being a single string value. As per [doc](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md) the format of "properties" isn't defined by ARM.

What should the schema look like to describe this response?

Reply:
Removed this API from the spec update for now.